recipes/libxc/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "libxc" %}
+{% set version = "4.3.3" %}
+{% set sha256 = "4dedcadbb5755b6da1836effabf0ad0d77a1b6fb6760d847a456d1ace9145faa" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  #url: http://www.tddft.org/programs/octopus/down.php?file=libxc/4.3.3/libxc-4.3.3.tar.gz
+  #sha256: 99ed7a31c7ae19e364d341e2d4a02b108a9b4f45c7f5c5926c6108612784aff0
+  url: https://gitlab.com/libxc/libxc/-/archive/{{ version }}/libxc-{{ version }}.tar.bz2
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  run_exports:
+    - {{ pin_subpackage('libxc') }}
```

Comment:
Is the ABI of libxc stable for all major versions? It's more common to be stable for minor versions (if at all) which requires one of:
```yaml
  - {{ pin_subpackage('libxc', max_pin='x.x') }}
  - {{ pin_subpackage('libxc', max_pin='x.x.x') }}
```

Reply:
Gotcha, wasn't sure what max_pin was doing. I'm not sure about libxc, but there was max_pin='x.x' in psi4's recipe so I restored that.

Reply:
That seems reasonable to me, for reference:
* If you build libxc 4.3.3:
  - `pin_subpackage('libxc')` becomes: `libxc >=4.3.3,<5`
  - `pin_subpackage('libxc', max_pin='x')` becomes: `libxc >=4.3.3,<5`
  - `pin_subpackage('libxc', max_pin='x.x')` becomes: `libxc >=4.3.3,<4.4`
  - `pin_subpackage('libxc', max_pin='x.x.x')` becomes: `libxc >=4.3.3,<4.3.4`
* If you build libxc 4.1.2:
  - `pin_subpackage('libxc')` becomes: `libxc >=4.1.2,<5`
  - `pin_subpackage('libxc', max_pin='x')` becomes: `libxc >=4.1.2,<5`
  - `pin_subpackage('libxc', max_pin='x.x')` becomes: `libxc >=4.1.2,<4.2`
  - `pin_subpackage('libxc', max_pin='x.x.x')` becomes: `libxc >=4.1.2,<4.1.3`